saltproc/input_schema.json

```diff
@@ -102,7 +98,7 @@
                                         "pattern": "^\\.\\/(.*)\\.xml$"}
                                 },
 
-                                "required": ["settings", "geometry", "materials", "chain_file"]
+                                "required": ["settings", "materials", "chain_file"]
```

Comment:
A geometry file is required, but we have an entirely separate object for that, `geo_files`

Reply:
I see. Ok, so I like the new geo_files object, but do you think it might be less intuitive that the geometry settings file path is not included in this input schema? All of the other paths are defined here. Would it potentially make it more difficult to read?

Reply:
It may be confusing at first, but that's just how SaltProc is structured. We could try to change that in the future. I think it would be more confusing for users to have two different geometry file specification parameters, because then which one are they supposed to use?

Reply:
Ok, that makes sense!